### DISCUSSION TITLE
home assistant notification "url"/"clickAction"

### DISCUSSION THREAD

**Discussion by vajonam** [blakeblackshear/frigate#6160] [Ask A Question] [answered]
I am trying to streamline the click/url action on an iOS HA notification as of now I have 

`"url": "/ccab4aaf_frigate-proxy/dashboard/`

this works fine. 

this opens up the frigate dashboard via the frigate-proxy addon, (frigate is hosted with GPU support on a different host). my question is that can I open up the /ccab4aaf_frigate-proxy/events/<event-id>/clip.mp4 screen inside the dashboard so the navigation is the correct context.  

I tried navigating to `"url": "/ccab4aaf_frigate-proxy/events/` that just opens up the HA iOS app and shows an empty screen, has this already been figured out?

**Comment by NickM-27** [comment, marked as answer]
the proxy does not support this, you would either need to use the ip:port or you need to use the integration to open to a specific event video

**Reply by vajonam** [reply]
Thanks for quick answer as always! saves me a bit of time.
